commands/publish/index.js

```diff
@@ -380,7 +380,7 @@ class PublishCommand extends Command {
           this.packagesToBeLicensed = [];
           this.logger.warn(
             "ENOLICENSE",
-            `Packages ${packagesWithoutLicense.map(pkg => pkg.name).join(", ")} are missing a license`
+            `Packages ${packagesWithoutLicense.map(pkg => pkg.name).join(", ")} are missing a root LICENSE file`
```

Comment:
That's certainly one way to resolve the warning, but it's not strictly what the warning is concerned about.

How about we add a second line to the warning instead, saying something like 
```
"To fix this, add a LICENSE.md file in the root of this repository. See https://choosealicense.com for additional guidance."
```

Reply:
Sure! I kind of figured something like this wouldn't be _exactly_ right since, it really isn't that much more helpful. But I wanted to start somewhere. I'll make the change, let me know if the style isn't right.

Reply:
Looks great, thanks!